src/commands/login.ts

```diff
@@ -0,0 +1,76 @@
+import { Command } from 'commander';
+import RestClient from '../client';
+import { HttpError } from '../client/errors';
+
+import { storeAuthToken, clearAuthStore } from '../config';
+import { API_URL_OPT } from '../options';
+import { success, error } from '../utils/logger';
+
+type Options = {
+  apiUrl: URL;
+  all: boolean;
+};
+
+async function loginHandler(this: Command, token: string) {
+  const opts: Options = this.optsWithGlobals();
+
+  let user: string;
+  try {
+    if (token.startsWith('tgpak_')) {
+      const info = await RestClient.getApiKeyInformation(opts.apiUrl, token);
+      const username = info.userFullName || info.username || '<unknown user>';
+      user = `${username} (project: ${info.projectName})`;
+    } else {
+      const info = await RestClient.getPersonalAccessTokenUser(
+        opts.apiUrl,
+        token
+      );
+      user = info.name || info.username;
+    }
+  } catch (e: any) {
+    if (e instanceof HttpError) {
+      error(
+        e.response.status >= 500
+          ? `The Tolgee server reported an error (${e.response.status} ${e.response.statusText}). Try again later.`
+          : 'Invalid token.'
+      );
+    } else {
+      throw e;
+    }
+
+    process.exit(1);
+  }
+
+  await storeAuthToken(opts.apiUrl, token);
+  success(`Logged in as ${user} on ${opts.apiUrl.hostname}. Welcome back!`);
```

Comment:
Here we should also print the id and name of the project when PAK provided.

Reply:
Project name is displayed, I guess the confusion arises from the variable name. See assignment line 22.

Reply:
Then rename the variable.